rpi-eeprom-config

```diff
@@ -98,22 +98,26 @@ class BootloaderImage(object):
 
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, \
-    description='RPI EEPROM config tool. \
+    description='Bootloader EEPROM configuration tool for the Raspberry Pi 4B. \
 \n\nThere are 3 operating modes: \
 \n\n1. Output the bootloader configuration stored in an EEPROM image file to \
 the screen (STDOUT): specify only the name of an EEPROM image file using the \
-EEPROM option. \
+\'eeprom\' option. \
 \n\n2. Output the bootloader configuration stored in an EEPROM image file to a \
-file: specify the EEPROM image file using the EEPROM option, and the output file\
-using the --out option. \
+file: specify the EEPROM image file using the \'eeprom\' option, and the output \
+file using the \'--out\' option.\
 \n\n3. Insert a new bootloader configuration into an EEPROM image file: specify \
-the EEPROM image file using the EEPROM option, the bootloader configuration file\
-using the --config option, and the name to use for the new file which will be a \
-combination of the original EEPROM image file, along with the new bootloader \
-configuration.')
+the source EEPROM image file using the \'eeprom\' option and the bootloader \
+configuration file using the \'--config\' option. A new file which is a \
+combination of the EEPROM image file, together with the new bootloader \
+configuration file will be created - specify its name using the \'--out\' option. \
+The new bootloader configuration will replace any configuration present in the \
+source EEPROM image.\
```

Comment:
This feels incredibly wordy for the `--help` output of a tool :confused: (or maybe that's just me?!)

If the "Insert a new bootloader configuration..." line was changed to "Inject a new bootloader configuration..." do you think it would still make sense if the "A new file which is a combination of the EEPROM image file, together with the new bootloader configuration file will be created" part was removed? :man_shrugging:

Reply:
It is wordy, but that was mainly because I was trying to avoid having to write a separate man page. You're also not really injecting something, since the new configuration is written to a new file. I'm happy with it as-is, on this one. I will leave it to the repo owner to decide which approach they want to go for.